.github/workflows/e2e-tests.yml

```diff
@@ -20,6 +20,12 @@ jobs:
                         script: |
                             composer install
                             vendor/bin/phpstan analyse
+                    -   os: ubuntu-latest
+                        php-version: '8.2'
+                        test-dir: tests-e2e/github/
+                        script: |
+                            composer install
+                            diff <(vendor/bin/phpstan analyse --error-format=raw --no-progress | sed "s|$(pwd)/||") expected-errors.txt
```

Comment:
looks great. another one for jira please - after we merged the PR and verified this works in the pipeline as expected

Reply:
I have fixed the github action, so the e2e tests should also work on forks now.
please rebase and we will see.

Reply:
@EmilMassey would be great you could do the jira e2e test